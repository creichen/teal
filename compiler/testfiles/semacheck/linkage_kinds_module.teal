fun f(x) = {
}

var v;