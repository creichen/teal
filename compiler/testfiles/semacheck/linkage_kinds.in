import linkage_kinds_module::f;
import linkage_kinds_module::v;

fun main () = {
    f := 5;
    v(10);
}