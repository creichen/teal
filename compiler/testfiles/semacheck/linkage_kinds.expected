testfiles/semacheck/linkage_kinds.in[5:5-5:5]: 'f' is declared as a function, but used as a variable.
testfiles/semacheck/linkage_kinds.in[6:5-6:9]: 'v' is declared as a variable, but used as a function.
